tests/test_environment_MatchEnv.py

```diff
@@ -11,6 +11,13 @@
 matplotlib.use("agg")
 
 
+def assert_ragged_array(arr):
+    """Assert the given array is a list of numpy arrays."""
+    assert type(arr) == list
```

Comment:
Always prefer isinstance over raw type checks unless you’re in a very specific circumstance (signac uses raw type checks for synced collections for performance reasons in one particular case).
```suggestion
    assert isinstance(arr, list)
```